flutter-idea/src/io/flutter/perf/FlutterWidgetPerf.java

```diff
@@ -5,6 +5,7 @@
  */
 package io.flutter.perf;
 
+import com.android.tools.idea.io.netty.util.collection.IntObjectHashMap;
```

Comment:
I'm guessing this import and class (here and elsewhere) might not be available in IntelliJ if the android plugin isn't available. Consider using the fastutil `Int2ObjectMap`. I believe that should be available in the base platform.

Reply:
`Int2ObjectMap` is abstract. Follow-on work could resolve this and not depend on com.android.tools.idea.io.netty.util.collection`.